popup.html

```diff
@@ -120,8 +120,11 @@
             <div v-show="info === 'export'">
                 <div id="security_warning">{{ i18n.export_info }}</div>
                 <a download="authenticator.json" v-bind:href="exportFile" id="download_backup">{{ i18n.download_backup }}</a>
-                <label id="upload_backup" for="import_file">{{ i18n.import_backup }}</label>
-                <input id="import_file" type="file" v-on:change="importFile($event)" accept="application/json">
+                <div v-if="navigator.userAgent.indexOf('Chrome') !== -1">
```

Comment:
it doesn't work on my side, Chrome opens import.html as well. No idea why...

Reply:
`navigator` is not set in Vue. It is a scope issue. In origin JS, `navigator` is in `window` namespace (`window.navigator`), but in Vue template, the scope is Vue instance, so `navigator` points to `authenticator.navigator` which is undefined. We have to wrap a function in ui to check UA.

Reply:
No wonder v-if was giving me such a hard time.